marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/mapping/getDocumentForTesting.api

```diff
@@ -0,0 +1,18 @@
+{
+    "functionName": "getDocumentForTesting",
```

Comment:
I know I proposed this, but seeing it here, I think it makes sense.

Reply:
Does or does not make sense?

Reply:
Does, I'm just patting myself on the back :)